<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SumWidgetForToolbar</class>
 <widget class="QWidget" name="SumWidgetForToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>28</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="spinBoxEnd">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>3</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox1">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>3</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>6</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>3</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>4</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
